5/10/2021 3:35:05 PM: getTableHeader-HomeModel: Error converting data type int to datetime.; SP: EXEC ACESch.ProduccionAceriaDemorasMesProc @dtFecha = 20210510, @nHorno = 2
5/10/2021 3:36:39 PM: getTableHeader-HomeModel: Error converting data type int to datetime.; SP: EXEC ACESch.ProduccionAceriaDemorasMesProc @dtFecha = 20210510, @nHorno = 2
5/11/2021 10:02:14 AM: getTableHeader-HomeModel: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server); SP: EXEC ACESch.ProduccionAceriaDemorasMesProc @dtFecha = '20210511', @nHorno = 2
5/11/2021 12:09:56 PM: getTableHeader-HomeModel: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server); SP: EXEC ACESch.ProduccionAceriaDemorasMesProc @dtFecha = '20210511', @nHorno = 2
5/25/2021 4:18:02 PM: getLogin-Account: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server); SP: EXEC ACESch.MVCUsuarioLogin_Sel @PsLoginUsuario = '903557', @PsContrasenaUsuario = '7148643'
5/25/2021 4:23:48 PM: HornoModel-getHornoFusion: Incorrect syntax near ','.; SP: EXEC AceSch.AceHornoFusionCmb @pnClaUbicacion = '7', @psValor = , @pnTipo = 1, @ModoSel = default , @pnModoSel = default, @pnBajasSn = default, @pnIncluirTodosSN = 0, @pnIncluirSinHornoSN = default, @pnDeaceroSN = default, @pnMostrarTodosSN = default, @pnFiltro = default, @pnClaSubAlmacen = default
7/12/2021 4:51:23 PM: getColadasPorRango-getCaracterizacion: Procedure or function 'ACECaracterizaChatarra_U' expects parameter '@PsNombrePCMod', which was not supplied.; SP: EXEC ACESch.ACECaracterizaChatarra_U @pnClaUbicacion = '7', @PsColadas = '182499,182500,182501', @PiHorno = 2, @PiMetodo = 1, @PNombrePCMod = '::1', @PClaUsuarioMod = 100012652, @PConsultar = 1
7/12/2021 5:07:08 PM: getColadasPorRango-getColadasPorRango: La colada inicial debe de ser menor a la colada final. Favor de corregir el n√∫mero de colada.; SP: EXEC ACESch.MVCColadasPorRangoFecha_Sel @pnClaUbicacion = '7', @ptFechaInicial = '20210712', @ptFechaFinal = '20210712', @pnHornoFusion = -1, @pnTurno39 = -1, @pnrdFechas = 0, @pnColadaInicial = 1822499, @pnColadaFinal = 182501
7/12/2021 6:58:29 PM: TurnoController-getTurno-System.NullReferenceException: Object reference not set to an instance of an object.
   at HGI.Models.Identities.LogEventosModel.getLogEventos(Object[] Args) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Models\Identities\LogEventosModel.cs:line 28
   at HGI.Controllers.Identities.LogEventosController.getLogEventos(Int32 IdModulo, String ClaveInt, String ClaveString) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Controllers\Identities\LogEventosController.cs:line 31
7/12/2021 6:58:29 PM: TurnoController-getTurno-System.NullReferenceException: Object reference not set to an instance of an object.
   at HGI.Models.Identities.LogEventosModel.getLogEventos(Object[] Args) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Models\Identities\LogEventosModel.cs:line 28
   at HGI.Controllers.Identities.LogEventosController.getLogEventos(Int32 IdModulo, String ClaveInt, String ClaveString) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Controllers\Identities\LogEventosController.cs:line 31
7/12/2021 6:58:39 PM: TurnoController-getTurno-System.NullReferenceException: Object reference not set to an instance of an object.
   at HGI.Models.Identities.LogEventosModel.getLogEventos(Object[] Args) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Models\Identities\LogEventosModel.cs:line 28
   at HGI.Controllers.Identities.LogEventosController.getLogEventos(Int32 IdModulo, String ClaveInt, String ClaveString) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Controllers\Identities\LogEventosController.cs:line 31
7/12/2021 6:58:39 PM: TurnoController-getTurno-System.NullReferenceException: Object reference not set to an instance of an object.
   at HGI.Models.Identities.LogEventosModel.getLogEventos(Object[] Args) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Models\Identities\LogEventosModel.cs:line 28
   at HGI.Controllers.Identities.LogEventosController.getLogEventos(Int32 IdModulo, String ClaveInt, String ClaveString) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Controllers\Identities\LogEventosController.cs:line 31
7/12/2021 6:58:43 PM: TurnoController-getTurno-System.NullReferenceException: Object reference not set to an instance of an object.
   at HGI.Models.Identities.LogEventosModel.getLogEventos(Object[] Args) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Models\Identities\LogEventosModel.cs:line 28
   at HGI.Controllers.Identities.LogEventosController.getLogEventos(Int32 IdModulo, String ClaveInt, String ClaveString) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Controllers\Identities\LogEventosController.cs:line 31
7/12/2021 6:58:44 PM: TurnoController-getTurno-System.NullReferenceException: Object reference not set to an instance of an object.
   at HGI.Models.Identities.LogEventosModel.getLogEventos(Object[] Args) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Models\Identities\LogEventosModel.cs:line 28
   at HGI.Controllers.Identities.LogEventosController.getLogEventos(Int32 IdModulo, String ClaveInt, String ClaveString) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Controllers\Identities\LogEventosController.cs:line 31
7/12/2021 6:58:46 PM: TurnoController-getTurno-System.NullReferenceException: Object reference not set to an instance of an object.
   at HGI.Models.Identities.LogEventosModel.getLogEventos(Object[] Args) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Models\Identities\LogEventosModel.cs:line 28
   at HGI.Controllers.Identities.LogEventosController.getLogEventos(Int32 IdModulo, String ClaveInt, String ClaveString) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Controllers\Identities\LogEventosController.cs:line 31
7/12/2021 6:58:47 PM: TurnoController-getTurno-System.NullReferenceException: Object reference not set to an instance of an object.
   at HGI.Models.Identities.LogEventosModel.getLogEventos(Object[] Args) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Models\Identities\LogEventosModel.cs:line 28
   at HGI.Controllers.Identities.LogEventosController.getLogEventos(Int32 IdModulo, String ClaveInt, String ClaveString) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Controllers\Identities\LogEventosController.cs:line 31
7/12/2021 6:59:25 PM: TurnoController-getTurno-System.NullReferenceException: Object reference not set to an instance of an object.
   at HGI.Models.Identities.LogEventosModel.getLogEventos(Object[] Args) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Models\Identities\LogEventosModel.cs:line 28
   at HGI.Controllers.Identities.LogEventosController.getLogEventos(Int32 IdModulo, String ClaveInt, String ClaveString) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Controllers\Identities\LogEventosController.cs:line 31
7/12/2021 6:59:29 PM: TurnoController-getTurno-System.NullReferenceException: Object reference not set to an instance of an object.
   at HGI.Models.Identities.LogEventosModel.getLogEventos(Object[] Args) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Models\Identities\LogEventosModel.cs:line 28
   at HGI.Controllers.Identities.LogEventosController.getLogEventos(Int32 IdModulo, String ClaveInt, String ClaveString) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Controllers\Identities\LogEventosController.cs:line 31
7/12/2021 7:14:52 PM: TurnoController-getTurno-System.NullReferenceException: Object reference not set to an instance of an object.
   at HGI.Models.Identities.LogEventosModel.getLogEventos(Object[] Args) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Models\Identities\LogEventosModel.cs:line 28
   at HGI.Controllers.Identities.LogEventosController.getLogEventos(Int32 IdModulo, String ClaveInt, String ClaveString) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Controllers\Identities\LogEventosController.cs:line 31
7/12/2021 7:14:55 PM: TurnoController-getTurno-System.NullReferenceException: Object reference not set to an instance of an object.
   at HGI.Models.Identities.LogEventosModel.getLogEventos(Object[] Args) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Models\Identities\LogEventosModel.cs:line 28
   at HGI.Controllers.Identities.LogEventosController.getLogEventos(Int32 IdModulo, String ClaveInt, String ClaveString) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Controllers\Identities\LogEventosController.cs:line 31
7/12/2021 7:15:06 PM: TurnoController-getTurno-System.NullReferenceException: Object reference not set to an instance of an object.
   at HGI.Models.Identities.LogEventosModel.getLogEventos(Object[] Args) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Models\Identities\LogEventosModel.cs:line 28
   at HGI.Controllers.Identities.LogEventosController.getLogEventos(Int32 IdModulo, String ClaveInt, String ClaveString) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Controllers\Identities\LogEventosController.cs:line 31
7/12/2021 7:15:22 PM: TurnoController-getTurno-System.NullReferenceException: Object reference not set to an instance of an object.
   at HGI.Models.Identities.LogEventosModel.getLogEventos(Object[] Args) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Models\Identities\LogEventosModel.cs:line 28
   at HGI.Controllers.Identities.LogEventosController.getLogEventos(Int32 IdModulo, String ClaveInt, String ClaveString) in c:\Code\Equipo - ER - Git\trunk\Code\AceriasMvc\AceriasMvc\Controllers\Identities\LogEventosController.cs:line 31
8/10/2021 3:32:45 PM: getConexiones-Account: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server); SP: EXEC [HGISch].[HGICadenasDeConexion_Sel] @PnClaUbicacion = 7, @PnClaSistema = 360
8/10/2021 3:32:46 PM: getConexiones-Account: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server); SP: EXEC [HGISch].[HGICadenasDeConexion_Sel] @PnClaUbicacion = 7, @PnClaSistema = 360
8/10/2021 3:33:12 PM: getConexiones-Account: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server); SP: EXEC [HGISch].[HGICadenasDeConexion_Sel] @PnClaUbicacion = 7, @PnClaSistema = 360
8/10/2021 3:33:13 PM: getConexiones-Account: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server); SP: EXEC [HGISch].[HGICadenasDeConexion_Sel] @PnClaUbicacion = 7, @PnClaSistema = 360
8/10/2021 3:33:16 PM: getConexiones-Account: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server); SP: EXEC [HGISch].[HGICadenasDeConexion_Sel] @PnClaUbicacion = 7, @PnClaSistema = 360
8/10/2021 3:33:21 PM: getConexiones-Account: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server); SP: EXEC [HGISch].[HGICadenasDeConexion_Sel] @PnClaUbicacion = 7, @PnClaSistema = 360
8/10/2021 3:33:55 PM: getConexiones-Account: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server); SP: EXEC [HGISch].[HGICadenasDeConexion_Sel] @PnClaUbicacion = 7, @PnClaSistema = 360
8/11/2021 12:07:48 PM: getConexiones-Account: Connection Timeout Expired.  The timeout period elapsed while attempting to consume the pre-login handshake acknowledgement.  This could be because the pre-login handshake failed or the server was unable to respond back in time.  The duration spent while attempting to connect to this server was - [Pre-Login] initialization=12679; handshake=9871; ; SP: EXEC [HGISch].[HGICadenasDeConexion_Sel] @PnClaUbicacion = 7, @PnClaSistema = 360
8/11/2021 12:11:43 PM: getPermisos-Account: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server); SP: EXEC dbo.HGIUsuarioPermisos_Sel @PnClaUbicacion = 7, @PsLoginUsuario = '903557', @PnClaSistema = 360
8/12/2021 11:13:43 AM: getConexiones-Account: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server); SP: EXEC [HGISch].[HGICadenasDeConexion_Sel] @PnClaUbicacion = 7, @PnClaSistema = 360
8/13/2021 11:51:10 AM: DensidadVolumen-getAgrupadoCargas: Procedure or function HgiChatarraDensidadVolumenDBGrupo_Rpt has too many arguments specified.; SP: EXEC [ACESch].[HgiChatarraDensidadVolumenDBGrupo_Rpt] @pnPlanta = '7', @pnHorno = '2', @pdFechaInicio = '20210813', @pdFechaFin = '20210813', @psProducto = 'CH-1,CH01,CH02,CH03,CH05,CH06,CH07,CH08,CH09,CH10,CH34,CH50,CH60'
8/16/2021 11:25:32 AM: DensidadVolumen-getAgrupadoCargas: Execution Timeout Expired.  The timeout period elapsed prior to completion of the operation or the server is not responding.; SP: EXEC [ACESch].[ACEHgiChatarraDensidadVolumenGrupo_Rpt] @pnPlanta = '7', @pnHorno = '2', @pdFechaInicio = '20210816', @pdFechaFin = '20210816', @psProducto = 'CH-1,CH01,CH02,CH03,CH05,CH06,CH07,CH08,CH09,CH10,CH34,CH50,CH60'
8/16/2021 1:55:07 PM: DensidadVolumen-getA1Diario: Divide by zero error encountered.
Warning: Null value is eliminated by an aggregate or other SET operation.; SP: EXEC [ACESch].[HgiChatarraDensidadVolumenDBGrupo_Rpt] @pnPlanta = '7', @pnHorno = '2'
8/16/2021 1:55:08 PM: DensidadVolumen-getAgrupadoCargas: Divide by zero error encountered.
Warning: Null value is eliminated by an aggregate or other SET operation.; SP: EXEC [ACESch].[ACEHgiChatarraDensidadVolumenGrupo_Rpt] @pnPlanta = '7', @pnHorno = '2', @pdFechaInicio = '20210816', @pdFechaFin = '20210816', @psProducto = 'CH-1,CH01,CH02,CH03,CH05,CH06,CH07,CH08,CH09,CH10,CH34,CH50,CH60'
8/16/2021 1:55:08 PM: DensidadVolumen-getAgrupadoSemanas: Divide by zero error encountered.
Warning: Null value is eliminated by an aggregate or other SET operation.
Warning: Null value is eliminated by an aggregate or other SET operation.
Warning: Null value is eliminated by an aggregate or other SET operation.
Warning: Null value is eliminated by an aggregate or other SET operation.
Warning: Null value is eliminated by an aggregate or other SET operation.
Warning: Null value is eliminated by an aggregate or other SET operation.
Warning: Null value is eliminated by an aggregate or other SET operation.; SP: EXEC [ACESch].[HgiChatarraDyVRptDia] @pnPlanta = '7', @pnHorno = '2', @pdFechaInicio = '20210816', @pdFechaFin = '20210816', @psProducto = 'CH-1,CH01,CH02,CH03,CH05,CH06,CH07,CH08,CH09,CH10,CH34,CH50,CH60', @PnEsReporting = '0'
8/16/2021 1:58:37 PM: DensidadVolumen-getA1Diario: Divide by zero error encountered.
Warning: Null value is eliminated by an aggregate or other SET operation.; SP: EXEC [ACESch].[HgiChatarraDensidadVolumenDBGrupo_Rpt] @pnPlanta = '7', @pnHorno = '2'
8/16/2021 1:58:38 PM: DensidadVolumen-getAgrupadoSemanas: Divide by zero error encountered.
Warning: Null value is eliminated by an aggregate or other SET operation.
Warning: Null value is eliminated by an aggregate or other SET operation.
Warning: Null value is eliminated by an aggregate or other SET operation.
Warning: Null value is eliminated by an aggregate or other SET operation.
Warning: Null value is eliminated by an aggregate or other SET operation.
Warning: Null value is eliminated by an aggregate or other SET operation.
Warning: Null value is eliminated by an aggregate or other SET operation.; SP: EXEC [ACESch].[HgiChatarraDyVRptDia] @pnPlanta = '7', @pnHorno = '2', @pdFechaInicio = '20210816', @pdFechaFin = '20210816', @psProducto = 'CH-1,CH01,CH02,CH03,CH05,CH06,CH07,CH08,CH09,CH10,CH34,CH50,CH60', @PnEsReporting = '0'
8/16/2021 1:59:14 PM: DensidadVolumen-getAgrupadoCargas: Execution Timeout Expired.  The timeout period elapsed prior to completion of the operation or the server is not responding.; SP: EXEC [ACESch].[ACEHgiChatarraDensidadVolumenGrupo_Rpt] @pnPlanta = '7', @pnHorno = '2', @pdFechaInicio = '20210816', @pdFechaFin = '20210816', @psProducto = 'CH-1,CH01,CH02,CH03,CH05,CH06,CH07,CH08,CH09,CH10,CH34,CH50,CH60'
8/16/2021 1:59:16 PM: DensidadVolumen-getAgrupadoSemanas: Divide by zero error encountered.
Warning: Null value is eliminated by an aggregate or other SET operation.
Warning: Null value is eliminated by an aggregate or other SET operation.
Warning: Null value is eliminated by an aggregate or other SET operation.
Warning: Null value is eliminated by an aggregate or other SET operation.
Warning: Null value is eliminated by an aggregate or other SET operation.
Warning: Null value is eliminated by an aggregate or other SET operation.
Warning: Null value is eliminated by an aggregate or other SET operation.; SP: EXEC [ACESch].[HgiChatarraDyVRptDia] @pnPlanta = '7', @pnHorno = '-1', @pdFechaInicio = '20210816', @pdFechaFin = '20210816', @psProducto = 'CH-1,CH01,CH02,CH03,CH05,CH06,CH07,CH08,CH09,CH10,CH34,CH50,CH60', @PnEsReporting = '0'
8/16/2021 1:59:17 PM: DensidadVolumen-getAgrupadoCargas: Divide by zero error encountered.
Warning: Null value is eliminated by an aggregate or other SET operation.; SP: EXEC [ACESch].[ACEHgiChatarraDensidadVolumenGrupo_Rpt] @pnPlanta = '7', @pnHorno = '-1', @pdFechaInicio = '20210816', @pdFechaFin = '20210816', @psProducto = 'CH-1,CH01,CH02,CH03,CH05,CH06,CH07,CH08,CH09,CH10,CH34,CH50,CH60'
8/16/2021 1:59:18 PM: DensidadVolumen-getA1Diario: Divide by zero error encountered.
Warning: Null value is eliminated by an aggregate or other SET operation.; SP: EXEC [ACESch].[HgiChatarraDensidadVolumenDBGrupo_Rpt] @pnPlanta = '7', @pnHorno = '-1'
8/23/2021 10:24:46 AM: getConexiones-Account: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server); SP: EXEC [HGISch].[HGICadenasDeConexion_Sel] @PnClaUbicacion = 7, @PnClaSistema = 360
8/25/2021 3:56:38 PM: getConexiones-Account: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server); SP: EXEC [HGISch].[HGICadenasDeConexion_Sel] @PnClaUbicacion = 7, @PnClaSistema = 360
8/25/2021 3:57:04 PM: getConexiones-Account: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server); SP: EXEC [HGISch].[HGICadenasDeConexion_Sel] @PnClaUbicacion = 7, @PnClaSistema = 360
8/25/2021 3:57:04 PM: getConexiones-Account: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server); SP: EXEC [HGISch].[HGICadenasDeConexion_Sel] @PnClaUbicacion = 7, @PnClaSistema = 360
8/25/2021 4:02:55 PM: getConexiones-Account: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server); SP: EXEC [HGISch].[HGICadenasDeConexion_Sel] @PnClaUbicacion = 7, @PnClaSistema = 360
8/25/2021 4:04:58 PM: getConexiones-Account: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server); SP: EXEC [HGISch].[HGICadenasDeConexion_Sel] @PnClaUbicacion = 7, @PnClaSistema = 360
